images/tidb-backup-manager/Dockerfile

```diff
@@ -1,30 +1,31 @@
-FROM pingcap/alpine-glibc:alpine-3.14
+FROM pingcap/alpine-glibc:alpine-3.14.3
```

Comment:
Can base image work in arm?

Reply:
yes, you can try docker buildx imagetools inspect pingcap/alpine-glibc:alpine-3.14.3